src-tauri/src/main.rs

```diff
@@ -56,14 +60,33 @@ fn main() {
         .expect("error while running tauri application");
 }
 
+#[tauri::command]
+async fn get_cloud_tests(state: tauri::State<'_, ApplicationState>, project: models::Project) -> Result<Vec<models::CloudTest>, String> {
+    let project_config = state
+        .project_manager
+        .load_project_config(project)
+        .map_err(|e| e.to_string())?;
+
+    if project_config.cloud_token.is_none() || project_config.cloud_project_id.is_none() {
+        return Err("missing cloud_token/cloud_project_id config".to_string());
+    }
+    let cloud_token = project_config.cloud_token.unwrap();
+    let cloud_project_id = project_config.cloud_project_id.unwrap();
+
+    let cloud_tests: Vec<models::CloudTest> = cloud::get_cloud_tests(&cloud_token, &cloud_project_id)
+        .await
+        .map_err(|e| {
+            e.to_string()
+        })?;
+    Ok(cloud_tests)
+}
+
 #[tauri::command]
 async fn close_splashscreen(window: Window) {
     // Close splashscreen
-    window
-        .get_window("splashscreen")
-        .expect("no window labeled 'splashscreen' found")
-        .close()
-        .unwrap();
+    if let Some(window) = window.get_window("splashscreen") {
+        window.close().unwrap();
+    }
```

Comment:
changed to check to see if the splashscreen is there before closing, encountered some race conditions or script reloads 🤷‍♂️